server/lib/s3backup/__init__.py

```diff
@@ -42,6 +42,13 @@ def put_object(self, Bucket, Key, Body, ContentMD5):
         return self.s3connector.put_object(Bucket=Bucket, Key=Key,
                 Body=Body, ContentMD5=ContentMD5)
 
+    def upload_fileobj(self, read_content, Bucket, Key, Config, ExtraArgs):
```

Comment:
If you make the above a method in `s3Connector` then these two routines are unnecessary.